Update the README of the expo test app with token info (#676)
twilio/twilio-voice-react-native: test/expo/README.md

   yarn install
   ```

>    If you're performing tests for an RC or a release, consider deleting the

Reviewer: Imo if we think this is best practice then I would remove the "consider"

  adb reverse tcp:8081 tcp:8081
  ```

## Tokens

By default, the test app will expect at least two token files in the `test/expo/constants/` folder.

- `test/expo/constants/e2e-tests-token.ts` for the basic call suite

- `test/expo/constants/e2e-preflightTest-token.ts` for the PreflightTest suite

Each file will need to export a token, like so:
```ts
export const token = '...';
```

> Optionally, to leverage the built-in Metro bundler feature for platform-specific files, consider having 4 files:

Reviewer: Imo if we think this is best practice then I would remove the "consider"

  adb reverse tcp:8081 tcp:8081
  ```

## Tokens

By default, the test app will expect at least two token files in the `test/expo/constants/` folder.

- `test/expo/constants/e2e-tests-token.ts` for the basic call suite

> - `test/expo/constants/e2e-preflightTest-token.ts` for the PreflightTest suite

Author: Leaving the original text as is.

Each file will need to export a token, like so:
```ts
export const token = '...';
```

Optionally, to leverage the built-in Metro bundler feature for platform-specific files:

- `test/expo/constants/e2e-tests-token.android.ts`
- `test/expo/constants/e2e-tests-token.ios.ts`
- `test/expo/constants/e2e-preflightTest-token.android.ts`
- `test/expo/constants/e2e-preflightTest-token.ios.ts`

Note that these `token` files contain secrets and should not be committed to the repo. They are ignored in the `test/expo/.gitignore` file.

> This way, you can have the Android tokens minted using FCM push credentials and the iOS tokens minted using APNS push credentials and have to do minimal work when switching between platforms.

Reviewer: ```suggestion
This way, you can have the Android tokens minted using FCM push credentials and the iOS tokens minted using APN push credentials, thus requiring minimal work when switching between platforms.
```

Author: APNs is the official term, I'd like to leave the S but i'll lowercase it.
